mobile/app/componentes/Mapa.js

```diff
@@ -58,10 +59,17 @@ export default function App() {
       const locaisMarkers = await getLocalizacoesRecentes();
       return locaisMarkers;
     } catch (error) {
-      console.log(
-        'Não foi possível buscar as localizações para os markers: ',
-        error
-      );
+      console.log('Não foi possível buscar as localizações para os markers: ', error);
+      return error;
+    }
+  };
+
+  const getLocalizacaoBackground = async () => {
+    try {
+      const localizacaoBackground = await enviarLocalizacaoBackground();
```

Comment:
Se você olhar o código da função enviarLocalizacaoBackground verá que ela não retorna qualquer valor.
Existe um return vazio, apenas para sair da função se um erro for detectado.
Então, se a função não retorna nada, não há o que ser recebido. 
Desta forma, esta const localizacaoBackground não terá valor algum.
Logo, também não faz sentido retornar tal const logo abaixo.